Data/Sys/GameSettings/G3L.ini

```diff
@@ -1,4 +1,4 @@
-# G3LE8P - Super Monkey Ball Adventures (TM)
+# G3LE8P - Super Monkey Ball Adventure
```

Comment:
i only changed the titles if they seemed incorrect, like they are in this game. If it causes issues or confusion, I'll be happy to revert it.